code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,303 @@
+	. = ..()
+	if(work_type == ABNORMALITY_WORK_REPRESSION)
+		DeliveryRepress(user,work_type,pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	Delivery(user,work_type,pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/SuccessEffect(mob/living/carbon/human/user, work_type, pe)
+	. = ..()
+	Delivery(user,work_type,pe)
+	return
+
+//what the fuck is this
+/obj/effect/letters_flow
+	name = "Storm of letters"
+	desc = "The storm rages on."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "letters"
+	layer = ABOVE_MOB_LAYER
+
+/obj/effect/temp_visual/maildecal
+	name = "A letter"
+	desc = "You have a bad feeling about this."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+	layer = BELOW_MOB_LAYER
+	duration = 10 SECONDS
+
+//x2 workspeed buff pro 2022 free hack free robux
+/datum/status_effect/workspeed_buff
+	id = "workspeed_buff"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/workspeed_buff
+	duration = 6 SECONDS
+
+/datum/status_effect/workspeed_buff/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	user.physiology.work_speed_mod *= 1.5
+
+/datum/status_effect/workspeed_buff/on_remove()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	user.physiology.work_speed_mod /= 1.5
+
+/atom/movable/screen/alert/status_effect/workspeed_buff
+	name = "A letter from the past"
+	desc = "Reading the letter made you want to try harder for your past friends."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+
+/datum/status_effect/nofear_buff
+	id = "nofear_buff"
+	status_type = STATUS_EFFECT_UNIQUE
+	alert_type = /atom/movable/screen/alert/status_effect/nofear_buff
+	duration = 6 SECONDS
+
+/datum/status_effect/nofear_buff/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	ADD_TRAIT(user, TRAIT_WORKFEAR_IMMUNE, TRAIT_GENERIC)
+	ADD_TRAIT(user, TRAIT_COMBATFEAR_IMMUNE, TRAIT_GENERIC)
+
+/datum/status_effect/nofear_buff/on_remove()
+	. = ..()
+	var/mob/living/carbon/human/user = owner
+	REMOVE_TRAIT(user,TRAIT_WORKFEAR_IMMUNE,TRAIT_GENERIC)
+	REMOVE_TRAIT(user,TRAIT_COMBATFEAR_IMMUNE,TRAIT_GENERIC)
+
+/atom/movable/screen/alert/status_effect/nofear_buff
+	name = "A letter from the present"
+	desc = "Reading the letter made you fearless."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "mail"
+
+//yes this is ripped off of paper and mermaid gift but fuck you.
+/obj/item/parcelself
+	name = "Parcel from your alternate self"
+	desc = "You should not see this."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "parcel"
+	inhand_icon_state = "parcel"
+	worn_icon_state = "parcel"
+	drop_sound = 'sound/items/handling/paper_drop.ogg'
+	pickup_sound =  'sound/items/handling/paper_pickup.ogg'
+	gender = NEUTER
+	color = "white"
+	var/mob/living/carbon/receiver = null
+
+//this is using the parcel. you either go insane or get special ego
```

Comment:
Comment seems incorrect? The only options here seem to be just getting special ego, and it's random whether it's an EGO suit or EGO weapon.